src/i18n/it.ts

```diff
@@ -0,0 +1,33 @@
+
+import { createLocale } from './utils'
+
+export default createLocale({
+  'PROCESSING' () {
+    return 'Caricamento…' as const
+  },
+  'BTN_ERROR' () {
+    return '❌Download Fallito!' as const
+  },
+
+  'DEPRECATION_NOTICE' (btnName: string) {
+    return `Attenzione!\nSembra che \`${btnName}\` non funzioni correttamente. Usa \`Parti Singole\`.\n(Potrebbe funzionare. Clicca \`OK\` per continuare.)` as const
+  },
+
+  'DOWNLOAD' <T extends string> (fileType: T) {
+    return `Scarica ${fileType}` as const
+  },
+  'DOWNLOAD_AUDIO' <T extends string> (fileType: T) {
+    return `Scarica ${fileType} Audio` as const
+  },
+
+  'IND_PARTS' () {
+    return 'Parti Singole' as const
+  },
+  'IND_PARTS_TOOLTIP' () {
+    return 'Scarica Parti Singole (BETA)' as const
+  },
+
```

Comment:
https://github.com/Xmader/musescore-downloader/commit/8945a538968b30a131b524b665c41a68da9d620f adds `VIEW_IN_LIBRESCORE`. Could you translate it also?

```suggestion

  'VIEW_IN_LIBRESCORE' () {
    return '...' as const
  },

```

Reply:
Hi,
I answer to your questions and doubts here. Just don't look at the translator. 
Based on the english version I made few changes in the spanish one and the italian one:

### Spanish
 ```
var es = createLocale({
        'PROCESSING'() {
            return 'Cargando…';
        },
        'BTN_ERROR'() {
            return '❌¡Descarga Fallida!';
        },
        'DEPRECATION_NOTICE'(btnName) {
            return `¡OBSOLETO!\nUtiliza \`${btnName}\` dentro de \`Partes Indivduales\` en su lugar.\n(Esto todavía puede funcionar. Pulsa \`Aceptar\` para continuar.)`;
        },
        'DOWNLOAD'(fileType) {
            return `Descargar ${fileType}`;
        },
        'DOWNLOAD_AUDIO'(fileType) {
            return `Descargar ${fileType} Audio`;
        },
        'IND_PARTS'() {
            return 'Partes individuales';
        },
        'IND_PARTS_TOOLTIP'() {
            return 'Descargar partes individuales (BETA)';
        },
        'FULL_SCORE'() {
            return 'Partitura Completa';
        },
	'VIEW_IN_LIBRESCORE' () {
    	    return 'Visualizar en {'Librescore'}' as const
  	},
    });
```

### Italian
```
var it = createLocale({
        'PROCESSING'() {
            return 'Caricamento…';
        },
        'BTN_ERROR'() {
            return '❌Download Fallito!';
        },
        'DEPRECATION_NOTICE'(btnName) {
            return `¡DEPRECATO!\nUtilizza invece \`${btnName}\` all'interno di \`Parti Indivduali\`.\n(Qusto potrebbe funzionare. Clicca \`Ok\` per continuare.)`;
        },
        'DOWNLOAD'(fileType) {
            return `Scarica ${fileType}`;
        },
        'DOWNLOAD_AUDIO'(fileType) {
            return `Scarica ${fileType} Audio`;
        },
        'IND_PARTS'() {
            return 'Parti Singole';
        },
        'IND_PARTS_TOOLTIP'() {
            return 'Scarica Parti Singole (BETA)';
        },
        'FULL_SCORE'() {
            return 'Spartito Completo';
        },
	'VIEW_IN_LIBRESCORE' () {
    	    return 'Visualizzare in {'Librescore'}' as const
  	},
    });
```

As regard _View_In_Librescore_ just remove _{'Librescore'}_ and use the right parameter.

Reply:
Thank you. The Spanish version is slightly out-dated.

Reply:
What's the difference between Spanish `Ver` and `Visualizar`?

Reply:
_Ver_ is similar to _mirar_ in spanish and they means _look/watch/see_;
_Visualizar_ means _visualize/view_

Reply:
Thanks. Shouldn't `Visualizzare` be also in its third-person singular present form `Visualizza` as `Scarica`?

Reply:
Yes you are right ... but, in both italilan and spanish do you want to use the present of the singular third-person?

**For example:** 
```
**INFINITIVE**: _Descargar partes individuales (BETA)_:
**Present**: _Descarga parte individuales (BETA)_
```

or do you want to use the infinitive?

They are both correct, is up to you :)

Reply:
Thanks. The infinitive form sounds good to me. How do I change them all to infinitive form?

Reply:
I changed the above comment with the infinitive version in both spanish and italian: [](https://github.com/Xmader/musescore-downloader/pull/103#discussion_r550871425)